No strudel guide (#594)
Start9Labs/documentation: site/source/service-guides/nostr/nostrudel.rst

.. _nostrudel:

> ====
> noStrudel
> ====

Reviewer: ```suggestion
========
noStrudel
========
```

> The NoStrudel is a web application designed specifically for exploring and understanding the intricacies of the Nostr decentralized social network protocol.

Reviewer: ```suggestion
noStrudel is a web app for exploring the nostr protocol.
```

    
**Available platforms**

- `StartOS <https://marketplace.start9.com/marketplace/nostrudel>`_

> - `noStrudel web client <https://nostrudel.ninja/>`_

Reviewer: ```suggestion
```

**Using noStrudel on StartOS**

> #. Install noStrudel service from Start9 registry on your server.

Reviewer: ```suggestion
#. Install the noStrudel service from the Start9 registry on your server.
```

    .. figure:: /_static/images/services/nostr/nostrudel-1.png
        :width: 50%
        :alt: install

#. Click "Start".

    .. figure:: /_static/images/services/nostr/nostrudel-2.png
        :width: 50%
        :alt: start

> #. When service is ready click launch UI to access your personal Iris web UI.

Reviewer: ```suggestion
#. When the service is ready click "Launch UI" to access noStrudel.
```

    .. figure:: /_static/images/services/nostr/nostrudel-3.png
        :width: 50%
        :alt: launch

> #. Choose how to setup your relays and create a new account or access an existing account by using a nip-07 browser extension. If creating a new account, be sure to securely back up the private key, preferably using Vaultwarden on your Start9 server. 

Reviewer: ```suggestion
#. Choose how to setup your relays and create a new account or access an existing account by using a nip-07 browser extension. If creating a new account, be sure to securely back up the private key, preferably using Vaultwarden on StartOS.
```

    .. figure:: /_static/images/services/nostr/nostrudel-4.png
        :width: 50%
        :alt: sign in

> #. To connect to your own private relay, go to Relayes -> App Relays, paste in your Nostr Relay Websocket URL (e.g. ws://address.onion) and click add. You can find this value in the Properties section of your Nostr RS Relay service.

Reviewer: ```suggestion
#. To connect to your own private relay, go to Relays -> App Relays, paste in your Nostr Relay Websocket URL (e.g. ws://address.onion) and click add. You can find this value in the Properties section of your Nostr RS Relay service.
```

    .. note:: 

>         If you plan to connect to your own private relay, you must use `Tor browser <https://www.torproject.org/>`_ or :ref:`run Tor<connecting-tor>` on your system and we suggest using Firefox which must be :ref:`configured to use Tor<connecting-ff>`.

Reviewer: ```suggestion
        If you plan to connect to your own private relay, you must use `Tor browser <https://www.torproject.org/>`_ or :ref:`run Tor<connecting-tor>` on your system.  We recommend using Firefox which must be :ref:`configured to use Tor<connecting-ff>`.
```
